app/resources/ephemera_projects/ephemera_project.rb

```diff
@@ -5,8 +5,14 @@ class EphemeraProject < Resource
   attribute :title, Valkyrie::Types::Set
   attribute :slug, Valkyrie::Types::Set
   attribute :top_language, Valkyrie::Types::Set
+  attribute :contributor_uids, Valkyrie::Types::Set
 
   def logical_structure
     []
   end
+
+  # Append contributor_uids to edit_users
+  def edit_users
+    self[:edit_users] | contributor_uids
```

Comment:
I was just saying yesterday that I'd never actually used `|` instead of `||` — is there a reason for `|` here?

Reply:
@escowles I just looked it up; it's defined on array as a set union https://ruby-doc.org/core-2.4.1/Array.html#method-i-7C

Reply:
@hackmastera 👍 thanks for looking that up, I was misreading it.  Now I see that it's a union of the edit users and contributor users.

Reply:
Sorry, a math background makes me reach for set operators. 😆 I can make this `+`

Reply:
@tpendragon the set operator deduplicates, which seems like a nice safeguard. Will `contributor_uids` never get saved to `edit_users`?

Reply:
Yes, I know it de-dups. I was going to just add `.uniq` at the end. :)